src/Providers/Maps/Leaflet/Features/Zoom.ts

```diff
@@ -40,24 +63,16 @@ namespace Provider.Maps.Leaflet.Feature {
 
         public refreshZoom(): void {
             if (this._map.features.zoom.isAutofit) {
```

Comment:
This method can now maybe be moved to the abstractMap class. What do you think @OS-giulianasilva?

Reply:
Nevermind!